diff -Naur a/pkg_testing_tools/use.py b/pkg_testing_tools/use.py
--- a/pkg_testing_tools/use.py	2024-02-17 19:29:23.000000000 +0200
+++ b/pkg_testing_tools/use.py	2024-03-19 10:28:05.345171023 +0200
@@ -53,13 +53,15 @@
         "l10n_",
         "kernel_",
         "abi_",
-        # "python_target_",
-        # "python_targets_",
-        # "ruby_targets_",
+        "llvm_slot_",
+        "python_target_",
+        "python_single_target_",
+        "python_targets_",
+        "ruby_targets_",
         "cpu_flags_",
     )
 
-    ignore_flags = set(["debug", "doc", "test", "selinux", "split-usr", "pic"])
+    ignore_flags = set(["debug", "profile", "test", "selinux", "split-usr", "systemd", "pic"])
 
     # some flags that *most* likely we shouldn't shuffle and test.
     for flag in flags:
