diff -Naur a/pkg_testing_tools/use.py b/pkg_testing_tools/use.py
--- a/pkg_testing_tools/use.py	2024-03-17 19:47:30.000000000 +0200
+++ b/pkg_testing_tools/use.py	2024-04-19 11:30:03.630304196 +0300
@@ -53,13 +53,13 @@
         "l10n_",
         "kernel_",
         "abi_",
-        # "python_target_",
-        # "python_targets_",
-        # "ruby_targets_",
+        "python_target_",
+        "python_targets_",
+        "ruby_targets_",
         "cpu_flags_",
     )
 
-    ignore_flags = set(["debug", "doc", "test", "selinux", "split-usr", "pic"])
+    ignore_flags = set(["debug", "test", "selinux", "split-usr", "pic"])
 
     # some flags that *most* likely we shouldn't shuffle and test.
     for flag in flags:
